packages/graphql-testing/src/client.ts

```diff
@@ -1,67 +0,0 @@
-import {
```

Comment:
Non blocking, just curious why this file is no longer needed

Reply:
This was added two years ago to stop certain events happening out of `react` `act`. We will need to reevaluate and figure out if this is even needed anymore. I am leaning towards not.